src/components/ChatMessage/ChatMessage.js

```diff
@@ -18,14 +20,21 @@ import {
   themeConfigDefault
 } from '../../props/defaults'
 
-import { ResponseRenderer } from '../ResponseRenderer'
+import { QueryOutput } from '../QueryOutput'
```

Comment:
If you are using an old version, it will be the same but if you upgrade then yeah it will need to be changed. All of these changes are in the docs now too so that hopefully will help